src/store/reducers/index.js

```diff
@@ -21,6 +21,21 @@ function findBlockIndex( blocks, clientId: string ) {
 	} );
 }
 
+/*
+ * insert block into blocks[], below / after block having clientIdAbove
+*/
+function insertBlock( blocks, block, clientIdAbove ) {
+	// TODO we need to set focused: true and search for the currently focused block and
+	// set that one to `focused: false`.
+	const insertionIndex = findBlockIndex( blocks, clientIdAbove );
+	if (insertionIndex === blocks.length - 1) {
```

Comment:
Do we really need the check? If it's just for guarding against the end of the array, I think `splice` takes care of that already so we don't need to. [See here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice).